testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/slavereconnect/DeploymentOverlayScenario.java

```diff
@@ -196,11 +196,17 @@ void testAfterReconnect(DomainClient masterClient, DomainClient slaveClient) thr
         Assert.assertEquals("initial", getOverrideProperty(slaveClient));
 
         //https://issues.jboss.org/browse/WFCORE-710 - even non-affected servers get the overlays
-        Assert.assertEquals(/*"running"*/ "reload-required",
+        Assert.assertEquals(/* "ok" */ "reload-required",
+                DomainTestUtils.executeForResult(
+                        Util.getReadAttributeOperation(SLAVE_ADDR.append(UnaffectedScenario.SERVER), "runtime-configuration-state"), slaveClient).asString());
+        Assert.assertEquals(/* "running" */ "reload-required",
                 DomainTestUtils.executeForResult(
                         Util.getReadAttributeOperation(SLAVE_ADDR.append(UnaffectedScenario.SERVER), "server-state"), slaveClient).asString());
         Assert.assertEquals("reload-required",
                 DomainTestUtils.executeForResult(
+                        Util.getReadAttributeOperation(SLAVE_ADDR.append(SERVER, "server-affected"), "runtime-configuration-state"), slaveClient).asString());
+        Assert.assertEquals("reload-required",
+                DomainTestUtils.executeForResult(
```

Comment:
We can probably get rid of this and the other 'server-state' checks for the slavereconnect scenarios, and just go with the new runtime-configuration-state ones?